x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/routes.tsx

```diff
@@ -5,12 +5,14 @@
  */
 
 import React from 'react';
-import { Route } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
 
 import { HostList } from './view';
 
-export const endpointHostsRoutes = () => [
-  <Route path="/:pageName(endpoint-hosts)">
-    <HostList />
-  </Route>,
-];
+export const EndpointHostsRoutes: React.FC = () => (
+  <Switch>
```

Comment:
Why was the `<Switch>` needed?

Reply:
I forget to add the not foundPage component

Reply:
thank you for that

Reply:
I will remove it, you are right there is no need

Reply:
nope change my mind, I added the NotFoundPage in it ;)

Reply:
:) 
Did you forget the RouteNotFound ?